src/CommunityToolkit.Maui.CameraView/Providers/CameraProvider.android.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Diagnostics;
+using Java.Lang;
+using Android.Content;
+using Android.Graphics;
+using Android.Hardware.Camera2;
+using Android.Hardware.Camera2.Params;
+using AndroidX.Core.Content;
+using AndroidX.Camera.Core;
+using AndroidX.Camera.Camera2.InterOp;
```

Comment:
Is it right that we are referring to Camera2 here? I think we wanted to migrate to CameraX where possible

Reply:
Some camera characteristics are only supported in Camera2, such as supported resolutions.